configs/catalog.json

```diff
@@ -236,6 +236,9 @@
               }
             }
           ]
+        },
+        {
+          "permission": "catalog:tenant:update"
```

Comment:
@mkanoor Does it make sense to include an `attributeFilter` for this permission, like the admin scopes on the others in this role?

Reply:
@gmcculloug 
Since we are using acts_as_tenant gem we are limited to working with a single tenant at a time. We were using the scopes when there were multiple instances and we need to split the instances based on admin, user and group. Tenants are created automatically in our database so we dont need the create and neither do we need the delete. I thought about catalog:tenant:read if we need to read the tenant settings for admin and user.